Library/Homebrew/utils/github/api.rb

```diff
@@ -21,6 +21,7 @@ module GitHub
         #{ALL_SCOPES_URL}
       #{Utils::Shell.set_variable_in_profile("HOMEBREW_GITHUB_API_TOKEN", "your_token_here")}
   EOS
+  GITHUB_PAT_REGEX = /^(?:[a-f0-9]{40}|gp1_[A-Za-z0-9_]{40,255})$/.freeze
```

Comment:
```suggestion
  GITHUB_PAT_REGEX = /^(?:[a-f0-9]{40}|gp1_\w{40,255})$/.freeze
```

or

```suggestion
  GITHUB_PERSONAL_ACCESS_TOKEN_REGEX = /^(?:[a-f0-9]{40}|gp1_\w{40,255})$/.freeze
```

because I'm not sure `PAT` is a widely recognised acronym

Reply:
Now this needs to be:

```suggestion
  GITHUB_PERSONAL_ACCESS_TOKEN_REGEX = /^(?:[a-f0-9]{40}|ghp_\w{36,251})$/.freeze
```